app/views/directives/resource-service-bindings.html

```diff
@@ -0,0 +1,28 @@
+<div class="expanded-section mar-bottom-xl" ng-if="$ctrl.showBindings">
+  <h3>Service Bindings</h3>
+  <overview-service-binding
+    ng-if="($ctrl.bindings | size)"
+    ng-repeat="binding in $ctrl.bindings track by (binding | uid)"
+    binding="binding"
+    service-classes="$ctrl.serviceClasses"
+    service-instances="$ctrl.serviceInstances"
+    secrets="$ctrl.secrets">
+  </overview-service-binding>
+  <div ng-if="($ctrl.bindableServiceInstances | size) && ({resource: 'bindings', group: 'servicecatalog.k8s.io'} | canI : 'create')">
+    <a href="" ng-click="$ctrl.createBinding()" role="button">
+      <span class="pficon pficon-add-circle-o"></span>
```

Comment:
If we add the circle here, we'll need to add it for other things like "Create Autoscaler" etc on these pages.

Needs an `aria-hidden="true"`.